docs/develop-resources/api-conventions/external-reference.md

```diff
@@ -0,0 +1,56 @@
+# Direct Resource External Reference Guide
+
+
+# TL;DR
+
+`status.externalRef` should be used as the GCP identifier of the Config Connector resource.
+
+
+# Basic Rules
+
+Config Connector should assign the `status.externalRef` value when it firstly acquires or creates the GCP resource.  
+
+Config Connector should use `status.externalRef` as the GCP identifier to retrieve, update and delete (if applicable) the resource.
+
+Config Connector should validate the `spec` corresponding identifiers (i.e. `spec.projectRef` `spec.location`, `spec.resourceID`) based on `status.externalRef`.
+
+The `status.externalRef` value can be used as [the resource reference `<kind>Ref.External`](./resource-reference.md#api-rule)
+
+Config Connector shall not write the Direct resources' resource ID back to the `spec.resourceID` field. Only users should configure the `spec` fields.[^1]
+
+![reconciliation](./image/externalref-reconcile.png)
+
+
+# API definition
+
+```
+type <Kind>Status struct {
+   /* A unique specifier for the <Kind> workerpool resource in GCP.*/
+   ExternLRef *string json:"externalRef,omitempty"
+   ...
+}
+```
+
+## `status.externalRef `format
```

Comment:
Nit: misplaced space

Reply:
Yeah, this is from "Docs™ to Markdown" extension, somehow it always misplaces the "`"